src/core_plugins/timelion/server/handlers/lib/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,33 @@
+const parseSheet = require('../parse_sheet');
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
+
+  it('does not allow whitespace used between two expressions', function () {
+    const data = ['.es() .es(404)'];
+    expect(function () {
+      parseSheet(data);
```

Comment:
This won't throw, `parseSheet` is only generating the AST. This statement generates an AST similar to the one below and is semantically valid.